pkg/commands/resolver.go

```diff
@@ -141,11 +141,6 @@ func makeBuilder(ctx context.Context, bo *options.BuildOptions) (*build.Caching,
 		return nil, err
 	}
 
-	if bo.ConcurrentBuilds == 0 {
-		bo.ConcurrentBuilds = runtime.GOMAXPROCS(0)
-	}
-	innerBuilder = build.NewLimiter(innerBuilder, bo.ConcurrentBuilds)
```

Comment:
is this now dead?

Reply:
It is, but `build.NewLimiter` is public so I'm somewhat reluctant to just delete it.